gsoc/2020/posts/jose_miguel_sanchez/post01.md

```diff
@@ -1,15 +1,39 @@
-# GSoC'20 RoboComp project: NAME OF THE PROJECT
+# GSoC'20 RoboComp project: Syntax-error highlighter for LearnBlock
  
 DATE
  
 ## About me
  
-SOME INFORMATION ABOUT YOU
-
+Hello! I'm José Miguel, a Computer Engineering student in Cáceres, Spain.
```

Comment:
Hello! I'm José Miguel, a Computer Engineering student at the University of Extremadura in Cáceres, Spain.